### PR TITLE
remove some Fortran unit_test stuff

### PR DESCRIPTION
**Pull request by zingale**
also eliminate some probins

### CODE REVIEW

**Comment by zingale** [comment]
tests: http://groot.astro.sunysb.edu/Microphysics/test-suite/gfortran/2022-01-02-004/index.html

File: unit_test/burn_cell/inputs_aprox13 [hunk 1]
```diff
@@ -1,2 +1,38 @@
+unit_test.run_prefix = "react_aprox13_"
 
-amr.probin_file = probin_aprox13
+unii_test.small_temp = 1.e5
```
Comment (maxpkatz) [inline unit_test/burn_cell/inputs_aprox13:3]: Typo

File: unit_test/burn_cell/inputs_aprox19 [hunk 2]
```diff
@@ -1,2 +1,43 @@
-do_cxx=1
-amr.probin_file = probin_aprox19
+unit_test.run_prefix = "react_aprox19_"
+
+unit_testsmall_temp = 1.e5
```
Comment (maxpkatz) [inline unit_test/burn_cell/inputs_aprox19:3]: Typo

File: unit_test/burn_cell/inputs_aprox13 (hunk 1 above)
Comment (zingale) [reply unit_test/burn_cell/inputs_aprox13]: fixed

File: unit_test/burn_cell/inputs_aprox19 (hunk 2 above)
Comment (zingale) [reply unit_test/burn_cell/inputs_aprox19]: fixed

**Review by maxpkatz — APPROVED** [review]

**Pull request merged**